### DISCUSSION TITLE
End of the line modules

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#69] [Q&A] [answered]
Add the "type" of module as "End of the line "
The module are ending the track like a stop plate. 
One way can be to add gable type "Non - end of Track", I added the None - end of track.
But i made a mistake  , Mr Fjällemark - please remove "non - end of track"

**Comment by fjallemark** [comment, marked as answer]
When you have a module like that, only add one exit, not two. Thanks for noticing about this. I will update the help content accordingly.
